tests/Integration/bootstrap.php

```diff
@@ -26,6 +26,7 @@
 define('_PS_IN_TEST_', true);
 define('_PS_ROOT_DIR_', dirname(__DIR__, 2));
 define('_PS_MODULE_DIR_', _PS_ROOT_DIR_ . '/tests/Resources/modules/');
+define('_PS_ALLOW_MULTI_STATEMENTS_QUERIES_', true);
```

Comment:
I would define it to false I think:
```suggestion
define('_PS_ALLOW_MULTI_STATEMENTS_QUERIES_', false);
```

Reply:
I just set this extra variable to make the tests work, because you set global to false in config/defines.inc.php

Reply:
Hmm, does it mean in integration tests we absolutely need this multistatement query? Shouldn't we fix the tests that do this to be compliant with this now-disabled feature?

Besides the integration tests were ok in 178x so it means it's a new test that was added in 8.0 so there shouldn't be too many places to fix right?

Reply:
>Hmm, does it mean in integration tests we absolutely need this multistatement query? Shouldn't we fix the tests that do this to be compliant with this now-disabled feature?

It's a good thing 👍 but it should not happen inside a Pull Request that aims to merge branches. If we put too much things in this PR it'll take way longer than needed.

Reply:
Hmm I disagree, this is actually the moment to make sure new things merged are integrated smartly, a merge PR shouldn't be a blunt merge all PR with quick fixes just to make ti work somehow That's when the differences between two branches collide and need to be adapted to new branch

AFAIK the `_PS_ALLOW_MULTI_STATEMENTS_QUERIES_` is supposed to ALWAYS be false on 80 it was kept configurable in 178 just to avoid breaking chances but in 80 it should be enforced to false, @atomiix correct me if I'm wrong By the way maybe this const should be removed in 80 and instead we can force the PDO/Doctrine configuration without relying on a configurable const?

Therefore if we're not even able to respect the security configuration we enforce for others in our own tests it means they are not correctly written

Reply:
@jolelievre I agree with you, I think tests should pass with the value set to `false` in this PR.
Sadly, nothing has been done in 8.0 to disable multi-statement queries in 8.0 so we should keep it configurable. We can force it for the v9.0 I guess.